### ISSUE TITLE
TLS 1.3

### ISSUE THREAD

**Issue by physkets** [OfflineIMAP/offlineimap#571]
With OpenSSL 1.1.1 [supporting](https://wiki.openssl.org/index.php/TLS1.3) TLS 1.3, are there plans to add a `tls1_3` option?

**Comment by nicolas33** [comment]
Yes, if contributors send patches. I think there's a patch to send to imaplib2 before.
